src/renderer/getDiForUnitTesting.tsx

```diff
@@ -59,7 +63,41 @@ export const getDiForUnitTesting = (
     di.override(isWindowsInjectable, () => false);
     di.override(isLinuxInjectable, () => false);
 
-    di.override(terminalSpawningPoolInjectable, () => new HTMLElement());
+    di.override(terminalSpawningPoolInjectable, () => document.createElement("div"));
+    di.override(createStorageInjectable, () => function <MockT>(key: string, defaultValue: MockT) {
```

Comment:
Deal breaker: overly complicated fake, and override at wrong level of abstraction. Should override file-system operations instead.